libs/media-player/src/audio-clip-player.cy.tsx

```diff
@@ -0,0 +1,18 @@
+import { AudioClipPlayer } from './lib/audio-clip-player';
+import { DefaultPlayPauseButton } from './lib/default-play-pause-button';
+
+describe('<AudioClipPlayer />', () => {
+    describe(`when there is a valid audio url`, () => {
+        const validAudioUrl = 'https://be.tsilhqotinlanguage.ca:3003/download?id=hello.mp3';
+
+        it('should render', () => {
```

Comment:
Add a test case that if you use the `AudioClipPlayer` as a child component and force a re-render with a new `url` this works.

Reply:
done